### PR TITLE
New API version for Microsoft.Resources stable/2022-09-01

### PR DESCRIPTION
**Pull request by meghanagupta95**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by meghanagupta95** [comment]
azp run

File: specification/resources/resource-manager/readme.md
Comment (leni-msft) [inline specification/resources/resource-manager/readme.md:138]: please add a new tag instead of modifying existing one

**Review by leniatgh — CHANGES_REQUESTED** [review]

File: specification/resources/resource-manager/readme.md
Comment (meghanagupta95) [reply specification/resources/resource-manager/readme.md:137]: Hi Lei, I'm not sure if I've made the correct change. Could you tell me what I'm missing here? Does the order of the tags matter?

**Comment by meghanagupta95** [comment]
Hi @leni-msft , pinging again. I think I made the appropriate edits but the checks are still failing. Could you help me figure out what I'm missing?

File: specification/resources/resource-manager/readme.md
Comment (leni-msft) [reply specification/resources/resource-manager/readme.md:137]: the order doesn't matter, but don't delete the `package-resources-2021-04` tag, because it may be referenced somewhere.

**Comment by leni-msft** [comment]
@meghanagupta95 please follow the guidance in https://github.com/Azure/azure-rest-api-specs/pull/20494#issuecomment-1232539462 to fix the CI errors

**Comment by mentat9** [comment]
@meghanagupta95 - Note the BreakingChangeReviewRequired label must be addressed before ARM review. See instructions in the PR comments.

**Comment by meghanagupta95** [comment]
@mentat9 @leni-msft  We need to add a 202 response for the Tags PUT/PATCH/DELETE APIs (Tags_CreateOrUpdateAtScope, Tags_UpdateAtScope, Tags_DeleteAtScope) We had not accounted for this earlier since we didn't expect RPs to return a 202 for modifying lightweight properties on resources. I believe this change was approved by the breaking change committee as well. 

The pipelines seem to be failing due to some changes that were already present in the previous API version, since I have copied the previous API version file and only made changes to the Tags_CreateOrUpdateAtScope, Tags_UpdateAtScope, Tags_DeleteAtScope APIs. Perhaps there were new validations added to the pipeline?

**Comment by mentat9** [comment]
> I believe this change was approved by the breaking change committee as well.

@meghanagupta95 - Normally, the breaking change process should add an approval label (Approved-BreakingChanges) to the PR upon approval (and remove the BreakingChangeReviewRequire label). Can you confirm with them?

Note: ARM review oncall this week is @rkmanda. Please remove the ARMChangesRequested label to queue your PR back to the (current) ARM reviewer.

**Comment by meghanagupta95** [comment]
Adding in current ARM reviewer @rkmanda

We were able to get approval for the breaking changes in this PR. Please approve the PR when possible.

**Comment by mentat9** [comment]
@meghanagupta95 - Your PR is not ready for ARM review.

Please create a new PR following this process (and check the box when you submit): 

**Adding a new API version
-[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).**

Ref: https://armwiki.azurewebsites.net/rp_onboarding/process/api_review.html#introduction.

Also, please study the materials mentioned in the submission form and attest to that by checking the box:

**[ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.**

Respond back by removing the ARMChangesRequested label.

**Comment by meghanagupta95** [comment]
New PR as requested at https://github.com/Azure/azure-rest-api-specs/pull/21203

**Comment by zizw123** [comment]
> New PR as requested at #21203

Can you close this PR if this is no longer valid? And get breaking change approved on the new PR as well. Thanks

**Comment by meghanagupta95** [comment]
> > New PR as requested at #21203
> 
> Can you close this PR if this is no longer valid? And get breaking change approved on the new PR as well. Thanks

@zizw123 do we need to get the breaking changes approved for the new PR even if both PRs have the same changes?

**Pull request closed without merging**
